libs/application-generic/src/services/calculate-delay/compute-job-wait-duration.service.ts

```diff
@@ -28,7 +30,7 @@ export class ComputeJobWaitDurationService {
     stepMetadata?: IWorkflowStepMetadata;
     payload: any;
     overrides: any;
-    chimeraResponse?: IChimeraDigestResponse | IChimeraDelayResponse;
+    chimeraResponse?: IChimeraDigestResponse | IDelayOutput;
```

Comment:
```suggestion
    response?: IDigestResponse | IDelayOutput;
```

Reply:
Yeah i thought to rename it as part of the digest work 😅

Reply:
done here anyways :)